app/views/browse/deployment.html

```diff
@@ -1,142 +1,266 @@
+              <small class="meta" ng-if="deployment">created <relative-timestamp timestamp="deployment.metadata.creationTimestamp"></relative-timestamp></small>
+            </h1>
+            <labels labels="deployment.metadata.labels" clickable="true" kind="deployments" title-kind="deployment configs" project-name="{{deployment.metadata.namespace}}" limit="3"></labels>
           </div>
-        </div><!-- /middle-header-->
-        <div class="middle-content">
-          <div class="container-fluid">
-            <div class="row" ng-if="deployment">
-              <div class="col-md-12">
-                <uib-tabset>
-                  <uib-tab active="selectedTab.details">
-                    <uib-tab-heading>Details</uib-tab-heading>
-                    <div class="resource-details">
-                      <ng-include src=" 'views/browse/_deployment-details.html' "></ng-include>
+        </div>
+      </div><!-- /middle-header-->
+      <div class="middle-content">
+        <div class="container-fluid">
+          <div class="row" ng-if="loaded">
+            <div class="col-md-12" ng-class="{ 'hide-tabs' : !deployment }">
+              <uib-tabset>
+                <uib-tab active="selectedTab.details">
+                  <uib-tab-heading>Details</uib-tab-heading>
+                  <div class="resource-details" ng-if="deployment">
+                    <div class="row">
+                      <div class="col-lg-6">
+                        <h3>Status</h3>
+                        <dl class="dl-horizontal left">
+                          <dt>Last Version:</dt>
+                          <dd>{{deployment | annotation : 'deployment.kubernetes.io/revision'}}</dd>
+                          <dt>Replicas:</dt>
+                          <dd>
+                            <replicas spec="deployment.spec.replicas"
+                                      disable-scaling="autoscalers.length"
+                                      scale-fn="scale(replicas)"
+                                      deployment="deployment"></replicas>
+                            <span ng-if="autoscalers.length">(autoscaled)</span>
+                            <div ng-if="deployment.status.updatedReplicas">
+                              {{deployment.status.updatedReplicas}} up to date
+                            </div>
+                            <div ng-if="deployment.status.availableReplicas || deployment.status.unavailableReplicas">
+                              <span ng-if="deployment.status.availableReplicas">{{deployment.status.availableReplicas}} available<span ng-if="deployment.status.unavailableReplicas">,</span></span>
+                              <span ng-if="deployment.status.unavailableReplicas">{{deployment.status.unavailableReplicas}} unavailable</span>
+                            </div>
+                          </dd>
+                          <dt>Paused:</dt>
+                          <dd>{{!!deployment.spec.paused}}</dd>
```

Comment:
don't love seeing true/false here. for now can you switch it to Yes/No.  I think long term I'd rather not show it at all when paused is false, and when paused is true highlight it some other way.